### PR TITLE
dotCMS/core#14327 Personas and View as in edit mode are enterprise feature

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/portlets/content-types/form/content-types-form.component.spec.ts [hunk 1]
```diff
@@ -213,7 +213,7 @@ describe('ContentTypesFormComponent', () => {
 
     // tslint:disable-next-line:max-line-length
     it('should set canSave property false when the form value is updated and then gets back to the original content (no community license)', () => {
-        spyOn(dotLicenseService, 'isEnterpriseLicense').and.returnValue(Observable.of(false));
+        spyOn(dotLicenseService, 'isEnterprise').and.returnValue(Observable.of(false));
```
Comment (fmontes) [inline src/app/portlets/content-types/form/content-types-form.component.spec.ts]: To be DRY with all this spies, we can group things into ```describes```

File: src/app/portlets/dot-edit-page/content/components/dot-edit-content-view-as-toolbar/dot-edit-content-view-as-toolbar.component.spec.ts [hunk 2]
```diff
@@ -107,100 +111,125 @@ describe('DotEditContentViewAsToolbarComponent', () => {
...
+            const whatsChanged: DebugElement = de.query(By.css('p-checkbox'));
+            expect(whatsChanged).toBe(null);
+        });
+
+        it('should have what\'s change checkbox', () => {
+            componentHost.pageState = new DotRenderedPageState(mockUser, JSON.parse(JSON.stringify(mockDotRenderedPage)), PageMode.PREVIEW);
+            fixtureHost.detectChanges();
+            const whatsChanged: DebugElement = de.query(By.css('p-checkbox'));
+            expect(whatsChanged).toBeTruthy();
+        });
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/content/components/dot-edit-content-view-as-toolbar/dot-edit-content-view-as-toolbar.component.spec.ts]: why these last two test are in the 'enterprise license' describe?

File: src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts [hunk 3]
```diff
@@ -13,18 +13,17 @@ export class DotPersonaSelectorComponent implements OnInit {
...
     @Output() selected = new EventEmitter<DotPersona>();
 
-    personasOptions: Observable<DotPersona[]>;
+    options: Observable<DotPersona[]>;
 
     constructor(private dotPersonasService: DotPersonasService, private dotMessageService: DotMessageService) {}
 
     ngOnInit() {
-        this.personasOptions = this.dotMessageService.getMessages(['modes.persona.no.persona'])
+        this.options = this.dotMessageService
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts]: pipe?

File: src/app/view/components/dot-device-selector/dot-device-selector.component.ts [hunk 4]
```diff
@@ -13,24 +13,23 @@ export class DotDeviceSelectorComponent implements OnInit {
...
-                            name: messages['editpage.viewas.default.device'],
-                            cssHeight: '',
-                            cssWidth: '',
-                            inode: '0'
-                        },
-                        ...devices
-                    ])
-            );
+
+        this.options = this.dotMessageService.getMessages(['editpage.viewas.default.device']).mergeMap((messages: string[]) =>
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-device-selector/dot-device-selector.component.ts]: pipe?

File: src/styles/primeng-theme/components/_dropdown.scss [hunk 5]
```diff
@@ -132,17 +125,14 @@ p-dropdown {
...
-                .ui-dropdown-trigger {
+                .ui-inputtext,
+                .ui-dropdown-trigger,
+                .ui-dropdown-label:hover {
                     color: $white;
-                    opacity: $field-disabled-opacity;
+                    opacity: 0.4;
                 }
 
-                .ui-dropdown-label {
```
Comment (hmoreras) [inline src/styles/primeng-theme/components/_dropdown.scss]: not needed ?

Comment (fmontes) [reply src/styles/primeng-theme/components/_dropdown.scss]: Move it to the top, keep it DRY

**Pull request merged**